massgov/mayflower-patternlab
# DP-5653 Conserve google maps api credits by optimizing our api calls

## Description
Update Google maps api calls to optimize usage and eliminate place details calls.

## Related Issue / Ticket

- [DP-5653 Conserve google maps api credits by optimizing our api calls](https://jira.state.ma.us/browse/DP-5653)

## Steps to Test


1. Visit https://kbentham.github.io/mayflower/?p=pages-location-listing
2. Verify location filtering works both through the auto complete and using a text string

## Screenshots
![old_request_details](https://user-images.githubusercontent.com/1236362/30712934-52b64ab0-9ed3-11e7-927b-41f2d4eeacbd.png)
![new_request_details](https://user-images.githubusercontent.com/1236362/30712945-5a32686e-9ed3-11e7-9a0b-1069353349bd.png)
 
If you look at the request details in the screenshot you will notice that we are no longer calling places details and are instead calling geocoder when making requests using the autocomplete.


## Additional Notes:

N/A

#### Impacted Areas in Application


* Location listing pages

#### @TODO
N/A

#### Today I learned...
